changelogs/unreleased/6367-server-status-fix.yml

```diff
@@ -0,0 +1,5 @@
+description: Fix server status throwing an error
+issue-nr: 6367
+change-type: patch
+destination-branches: [master, iso8]
+
```

Comment:
Once in a while, it doesn't hurt to add a section to show the description :)

Reply:
I opted not to add description to issues that did not went into user facing production, I don't know if that has sense, but why user should read about issue resolved that was was introducent in the iteration, from his perspercive there was no issue in the first place :) THat's why I recently don't add descriptions :)

Reply:
let's rediscuss that on the planning meeting then